src/styles/_base_utility.scss

```diff
@@ -59,6 +59,10 @@
   left: 0;
   width: 100%;
 
+  .row {
```

Comment:
this class feels like an escape hatch -> I'm alright with it, I'd just prefer if the mixin were used directly on the relevant stylesheet on a a specific row class for that container.. This feels like an escape hatch